### PR TITLE
[KIECLOUD-466] - Avail the possibility to use the ubi8-mininal as bas…

### PR DESCRIPTION
**Pull request by Kaitou786**
…e image for RHPAM and RHDM images

See: https://issues.redhat.com/browse/KIECLOUD-466

Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by msmagnanijr — DISMISSED** [review]

File: businesscentral-monitoring/image.yaml [hunk 1]
```diff
@@ -137,6 +138,7 @@ modules:
           - name: jboss-kie-workbench
 packages:
       content_sets_file: content_sets.yml
+      manager: microdnf
       install:
           - hostname
```
Comment (spolti) [inline businesscentral-monitoring/image.yaml:143]: remove the hostname package from the image.yaml as it is being installed by the common.dependencies module.

File: businesscentral/image.yaml [hunk 2]
```diff
@@ -187,6 +188,7 @@ modules:
           - name: jboss-kie-workbench
 packages:
       content_sets_file: content_sets.yml
+      manager: microdnf
       install:
           - hostname
```
Comment (spolti) [inline businesscentral/image.yaml:193]: same here.

File: controller/image.yaml [hunk 3]
```diff
@@ -120,6 +121,7 @@ modules:
           - name: jboss-kie-controller
 packages:
       content_sets_file: content_sets.yml
+      manager: microdnf
       install:
           - hostname
```
Comment (spolti) [inline controller/image.yaml:126]: same here.

File: kieserver/image.yaml [hunk 4]
```diff
@@ -264,6 +265,7 @@ modules:
           - name: jboss-kie-kieserver
 packages:
       content_sets_file: content_sets.yml
+      manager: microdnf
       install:
           - hostname
```
Comment (spolti) [inline kieserver/image.yaml:270]: same here.

File: smartrouter/image.yaml [hunk 5]
```diff
@@ -112,6 +113,7 @@ modules:
           - name: jboss-kie-smartrouter
 packages:
       content_sets_file: content_sets.yml
+      manager: microdnf
       install:
           - hostname
```
Comment (spolti) [inline smartrouter/image.yaml:118]: same here.

**Review by spolti — CHANGES_REQUESTED** [review]

**Review by spolti — APPROVED** [review]

**Pull request merged**